content/en/contribute/medic/product-development-process/ux-research-repo/key-concepts.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Key Concepts"
+linkTitle: "Key Concepts"
+weight: 1
+description: >
+  Baserow Key Concepts
+---
+
+There are a number of Baserow key concepts to understand before diving in. These are specific to how we’ve chosen to implement the repository.
+
+## Interview Subject
+
+This is the person that was interviewed or otherwise provided information/feedback. 
+
+**Example**: *Julius Nyerere*
+
+## Organization
+
+The company that the “Interview Subject” belongs to or associates with.
+
+**Example**: *MoH Furahi*
+
+## Samples
+
+These are links to, and metadata about, the “Evidence”. These are typically audio or video recordings stored on google drive, but can also be links to PDFs, forum posts, etc..
+
+**Example**: *Audio recording of a generative interview with Julius on 9-June-2021*
+
+## Nuggets
+
+An observation gathered through research or an atomic unit of research insight.
+
+**Example**: *Julius tapped on the X icon many times before the window closed. He seemed frustrated.*
+
```

Comment:
```suggestion

## Leads

Product suggestions or requests that did _not_ necessarily come directly from a product trio member interviewing or observing a user. These are often taken from forum posts or other 3rd party means.

**Example**: *Forum Post: It would be nice if the CHT used comic sans font* 

```